### PR TITLE
[Python] Fix typo in serializer exception

### PR DESCRIPTION
**Pull request by rberenguel**
## What changes were proposed in this pull request?

Fix typo in exception raised in Python serializer

## How was this patch tested?

No code changes

Please review http://spark.apache.org/contributing.html before opening a pull request.

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
@rberenguel, it's okay but mind taking another look around here and see if there are more typos while we are here? I am pretty sure there are more.

**Comment by rberenguel** [comment]
@HyukjinKwon sure, no problem. I'm reading pieces of the source for a presentation on PySpark and this jumped quickly. I'll try to collect a few more 👍

**Comment by HyukjinKwon** [comment]
ok to test

**Comment by HyukjinKwon** [comment]
ok to test

File: python/pyspark/serializers.py [hunk 1]
```diff
@@ -461,7 +462,7 @@ def dumps(self, obj):
         return obj
 
 
-# Hook namedtuple, make it picklable
+# Hack namedtuple, make it picklable
```
Comment (HyukjinKwon) [inline python/pyspark/serializers.py]: Oh, haha. Hook and Hack.

**Comment by HyukjinKwon** [comment]
Merged to master and branch-2.3.

**Pull request closed without merging**